build_library/grub_install.sh

```diff
@@ -110,6 +116,21 @@ info "Installing GRUB ${FLAGS_target} in ${FLAGS_disk_image##*/}"
 LOOP_DEV=$(sudo losetup --find --show --partscan "${FLAGS_disk_image}")
 ESP_DIR=$(mktemp --directory)
 
+# Hack available for doing the build in a container; see the LOOP_NO_UDEV
+# comments above.
+if [[ ${LOOP_NO_UDEV} -eq 1 ]]; then
+    echo "LOOP_NO_UDEV enabled, creating device nodes directly"
+    LOOPNUM=$(echo ${LOOP_DEV} | sed -e 's,/dev/loop,,')
```

Comment:
These scripts are already full of bashisms, so we might as well cut out unnecessary processes: `LOOPNUM=${LOOP_DEV#/dev/loop}`